Formula/mgis.rb

```diff
@@ -0,0 +1,40 @@
+class Mgis < Formula
+  desc "Provide tools to handle MFront generic interface behaviours"
+  homepage "https://thelfer.github.io/mgis/web/index.html"
+  url "https://github.com/thelfer/MFrontGenericInterfaceSupport/archive/refs/tags/MFrontGenericInterfaceSupport-2.1.tar.gz"
+  sha256 "f5b556aab130da0c423f395fe4c35d6bf509dd8fc958242f2e37ea788464aea9"
+  license "GPL-1.0-or-later"
```

Comment:
is there any reference for this? looks like it should LGPL3??

```
/*!
 * \file   BehaviourData.cxx
 * \brief
 * \author Thomas Helfer
 * \date   02/08/2018
 * \copyright (C) Copyright Thomas Helfer 2018.
 * Use, modification and distribution are subject
 * to one of the following licences:
 * - GNU Lesser General Public License (LGPL), Version 3.0. (See accompanying
 *   file LGPL-3.0.txt)
 * - CECILL-C,  Version 1.0 (See accompanying files
 *   CeCILL-C_V1-en.txt and CeCILL-C_V1-fr.txt).
 */
```

Reply:
yes, you're right